editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.tsx

```diff
@@ -79,6 +80,7 @@ export function addFakeSpyEntry(
       },
       element,
     ),
+    conditionalValue: conditionalValue,
```

Comment:
shouldn't this be null? the fake entry is the -child- of the ternary, right?

Reply:
oh sorry disregard, the fake entry is the one true entry for the ternary :D we should probably not call it fake. I confused it with `addConditionalAlternative` which _is_ the fake entry for the path-non-taken